### ISSUE TITLE
Empty search looks for "undefined"

### ISSUE THREAD

**Issue by fgaribay-nearsoft** [philc/vimium#3154]
Minor edge-case.

## Summary
Searching for an empty string will look for "undefined".

### Versions
```
Google Chrome | 69.0.3497.100 (Official Build) (64-bit)
OS | Mac OS X
Vimium | 1.64
```

## Steps to replicate
1. Type "**undefined**" in any text box (e.g. GitHub).
2. Unselect the text-box by clicking elsewhere.
3. Press **n**.

**Current result:** The word "undefined" will be highlighted, marking it as _found_.
**Expected result:** No word should be _found_ since we have not typed any search string.

### Exceptions
This behavior will not occur if another search string was already typed (_duh_), even if it was deleted; unless the search string was completely removed by typing slash again (`/`), followed by escape (`ESC`).

**Comment by smblott-github** [comment]
Are you sure you haven't stumbled upon a feature?

`/` has a history, and `n` has a memory.  Was `undefined` the last thing you searched for?

**Comment by gdh1995** [comment]
@smblott-github Confirmed - This issue occurs when I remove the 'findModeRawQueryList' item from `chrome.storage.local` and then do the steps above.

I think a simple check is enough.

**Comment by smblott-github** [comment]
Thanks, @gdh1995.

Fixed in 7c77b9bef6720283b04f6253b5e3cec0c7612b15.
